released/packages/coq-freespec-core/coq-freespec-core.0.3/opam

```diff
@@ -0,0 +1,47 @@
+opam-version: "2.0"
+maintainer: "Thomas Letan <lthms@soap.coffee>"
+
+homepage: "https://github.com/lthms/FreeSpec"
+dev-repo: "git+https://github.com/lthms/FreeSpec.git"
+bug-reports: "https://github.com/lthms/FreeSpec.git/issues"
+doc: "https://lthms.github.io/FreeSpec"
+license: "MPL-2.0"
+
+synopsis: "A framework for implementing and certifying impure computations in Coq"
+description: """
+FreeSpec is a framework for the Coq proof assistant which allows to
+implement and specify impure computations. This is the core of the
+framework: it provides the foundation of the formalism, based on the
+freer monad, the reasoning theory and tactics to automate the
+reasoning.
+"""
+
+build: [
+  ["patch" "-p1" "-i" "patches/opam-builds.patch"]
+  ["dune" "build" "-p" name "-j" jobs]
+]
+
+depends: [
+  "ocaml"
+  "dune" {>= "2.5"}
+  "coq" {>= "8.12" & < "8.14~"}
+  "coq-ext-lib" {>= "0.11.2"}
+]
+
+tags: [
+  "keyword:effects"
+  "keyword:freer"
+  "keyword:program logic"
+  "category:Mathematics/Category Theory"
+  "logpath:FreeSpec.Core"
```

Comment:
Ideally there should be a `date:yyyy-mm-dd` field too.

Reply:
Should be good